content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,265 @@
+===================
+Track and bill time
+===================
+
+*Helpdesk* provides teams with the ability to track the amount of hours spent working on a ticket,
+and to bill a customer for that time. Through integrations with the *Sales*, *Timesheets*,
+*Project*, and *Accounting* applications, customers can be charged once the work is completed, or
+before it has even begun.
+
+.. warning::
+   Since the *Track & Bill Time* features require integration with other applications, enabling them
+   may result in the installation of additional modules or applications.
+
+   Installing a new application on a *One-App-Free* database triggers a 15-day trial. At the end of
+   the trial, if a paid subscription has not been added to the database, it will no longer be active
+   or accessible.
+
+Track and bill time features configuration
+==========================================
+
+Before a customer can be invoiced for support services, the *Track & Bill Time* features **must** be
+enabled on each *Helpdesk* team individually.
+
+Enable track and bill time on a team
+------------------------------------
+
+To view and enable the *Track & Bill Time* features on a *Helpdesk* team, first navigate to
+:menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams`. Then, select a team from the
+list, or create a :doc:`new one <../overview/getting_started>`. This reveals a team's settings page.
```

Comment:
This is a nitpick 😄 please use meaningful/contextual label names for links. This helps specify exactly which section you are referring to as well as show the reader where the link will take them.
```suggestion
list, or create a :doc:`new Helpdesk team <../overview/getting_started>`. This reveals a team's
settings page.
```